pkg/internal/launcher/launcher.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of the kiagnose project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package launcher
+
+import (
+	"context"
+	"errors"
+	"strings"
+	"time"
+
+	"github.com/kiagnose/kubevirt-rt-checkup/pkg/internal/status"
+)
+
+type checkup interface {
```

Comment:
isn't launcher a library we want to import from kiagnose?

Reply:
I am not sure if the original launcher will work for this checkup, because there may be a need to report more than twice.

Reply:
I don't follow